bench/locli/src/Cardano/Util.hs

```diff
@@ -32,16 +31,20 @@ import Cardano.Prelude
 
 #if __GLASGOW_HASKELL__ < 902
 -- This is a GHC module ...
-import Util
-#else
+import Util                      hiding (fst3, snd3)
+#elif __GLASGOW_HASKELL__ < 906
```

Comment:
```suggestion
#elif __GLASGOW_HASKELL__ < 960
```

Reply:
That would mean GHC 9.60 and not GHC 9.6 https://downloads.haskell.org/ghc/latest/docs/users_guide/phases.html?highlight=__glasgow_haskell__#standard-cpp-macros